### PR TITLE
🔥hotfix: mistake from #47

### PR DESCRIPTION
**Pull request by michael-je**
hotfix: mistake from #47

Potentially solves #50?

### CODE REVIEW

**Comment by michael-je** [comment]
I won't be home tonight so feel free to change/merge this yourselves.

File: bot.py [hunk 1]
```diff
@@ -451,7 +451,8 @@ async def attempt_disconnect(self):
         if self.voice_client.is_playing():
             return
 
-        if self.last_played_time < self.DISCONNECT_TIMER_SECONDS:
+        time_since_last_played = time.time() - self.last_played_time
+        if time_since_last_played < self.DISCONNECT_TIMER_SECONDS:
```
Comment (morgaesis) [inline bot.py:455]:
```suggestion
        if time.time() - self.last_played_time < self.DISCONNECT_TIMER_SECONDS:
```

Comment (morgaesis) [reply bot.py:455]: Oneliner :hot_face: 

Slightly less readable, but doesn' make a new "unnecessary" variable.

**Review by morgaesis — APPROVED** [review]
LGTM :+1:

File: bot.py (hunk 1 above)
Comment (michael-je) [reply bot.py:455]: Yeah I guess this is clear enough

**Review by gardarandri — APPROVED** [review]

**Pull request merged**